plugins/Referrers/API.php

```diff
@@ -143,7 +144,14 @@ public function getAll($idSite, $period, $date, $segment = false)
         Piwik::checkUserHasViewAccess($idSite);
 
         $this->checkSingleSite($idSite, 'getAll');
-        $dataTable = $this->getReferrerType($idSite, $period, $date, $segment, $typeReferrer = false, $idSubtable = false, $expanded = true);
+        $dataTable = Request::processRequest('Referrers.getReferrerType', [
+            'idSite' => $idSite,
+            'period' => $period,
+            'date' => $date,
+            'segment' => $segment,
+            'expanded' => true,
+            'disable_queued_filters' => true
```

Comment:
does the generic filter also need to be disabled? or is no generic filter applied cause no filter is set? Looks otherwise good to merge if all the tests pass

Reply:
actually there shouldn't be any applied. but shouldn't hurt to add the param nevertheless